packages/pytest-simcore/src/pytest_simcore/docker_registry.py

```diff
@@ -285,3 +293,17 @@ def dy_static_file_server_dynamic_sidecar_compose_spec_service(
         docker_registry,
         node_meta_schema,
     )
+
+
+@pytest.fixture
+def remove_images_from_host() -> Callable[[list[str]], Awaitable[None]]:
+    async def _cleaner(images: list[str]) -> None:
+        print(f"--> Removing {images}")

```

Comment:
MINOR: Can use the new `log_context` here instead of the `print`.

Reply:
actually I am wondering why the regular one is not also upgraded with the same options

Reply:
now I know...